frontend/public/components/notification-drawer.tsx

```diff
@@ -0,0 +1,262 @@
+import * as _ from 'lodash';
```

Comment:
So at this point I would think we want to add new components to `frontend/packages/console-app/components` rather than `public/components`.  Thinking `console-app` instead of `console-shared` as this is an app wide singleton component.